src/main/scala/dpla/ingestion3/mappers/providers/PaExtractor.scala

```diff
@@ -5,13 +5,15 @@ import java.net.URI
 import dpla.ingestion3.mappers.xml.XmlExtractionUtils
 import dpla.ingestion3.model.DplaMapData.{ExactlyOne, ZeroToOne}
 import dpla.ingestion3.model._
+import dpla.ingestion3.utils.Utils
+import org.json4s.JsonDSL._
 
 import scala.util.Try
 import scala.xml._
 
 class PaExtractor(rawData: String, shortName: String) extends Extractor with XmlExtractionUtils {
 
-  implicit val xml: NodeSeq = XML.loadString(rawData)
+  implicit val xml = XML.loadString(rawData)
```

Comment:
I get yelled at by IDEA when there's no type tag here because the field's not `private`. I assume you did this because you need a `Node` instead of a `NodeSeq` in order to format it in line 69?

Reply:
Exactly but the formatting still isn't working so I can just change this back. Part of the problem is with the CQA app

Reply:
I could also change it to `implicit val xml: Elem` I think that will get us both what we want atm.

Reply:
@mdellabitta going with last option and then going to merge

Reply:
👍